### PR TITLE
Ensure sidekiq jobs are retried.

### PR DESCRIPTION
**Pull request by deepfryed**
Default option in sidekiq sets retry => true [1]. Adding a matcher in exq to retry such jobs.

[1] https://github.com/mperham/sidekiq/blob/master/lib/sidekiq.rb#L41

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -17,7 +17,7 @@ config :exq,
   redis_timeout: 5000,
   genserver_timeout: 5000,
   test_with_local_redis: true,
-  max_retries: 0,
+  max_retries: 1,
```
Comment (akira) [inline config/test.exs]: Rather than changing config value for all tests, can we just change for your specific test?

Comment (deepfryed) [reply config/test.exs]: Done

**Comment by akira** [comment]
👍 Thanks for the PR!

**Comment by deepfryed** [comment]
cheers, please give me a shout out when you bump version on hex.pm.  thanks!

**Comment by akira** [comment]
@deepfryed just bumped

**Comment by deepfryed** [comment]
thanks 🚀

**Pull request merged**